src/devices/Vcnl4040/Internal/PsConf1Register.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.I2c;
+using Iot.Device.Vcnl4040.Definitions;
+
+namespace Iot.Device.Vcnl4040.Internal
+{
+    /// <summary>
+    /// PS configuration register 1
+    /// Command code / address: 0x03 (LSB)
+    /// Documentation: datasheet (Rev. 1.7, 04-Nov-2020 9 Document Number: 84274).
+    /// </summary>
+    internal class PsConf1Register : Register
+    {
+        private static readonly byte PsDutyMask = 0b1100_0000;
+        private static readonly byte PsPersMask = 0b0011_0000;
+        private static readonly byte PsItMask = 0b0000_1110;
+        private static readonly byte PsSdMask = 0b0000_0001;
+        private bool _pPsDutyChanged = false;
+        private bool _psPersChanged = false;
+        private bool _psItChanged = false;
+        private bool _psSdChanged = false;
+        private PsDuty _psDuty = PsDuty.Duty40;
+        private PsInterruptPersistence _psPers = PsInterruptPersistence.Persistence1;
+        private PsIntegrationTime _psIt = PsIntegrationTime.Time1_0;
+        private PowerState _psSd = PowerState.PowerOff;
+
+        /// <summary>
+        /// Gets or sets the duty ratio (PS_Duty).
+        /// </summary>
+        public PsDuty PsDuty
+        {
+            get => _psDuty;
+            set
+            {
+                _psDuty = value;
+                _pPsDutyChanged = true;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the interrupt persistence (PS_PERS).
+        /// </summary>
+        public PsInterruptPersistence PsPers
```

Comment:
for those as well, will be better with more descriptive naming.

Reply:
We'll keep it as is, as discussed for AlsConfRegister.